src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> lines = readFile(fromFileName);
+        List<String> reportLines = processLines(lines);
+        writeFile(reportLines, toFileName);
+    }
+
+    public void writeFile(List<String> lines, String toFileName) {
```

Comment:
```suggestion
    private void writeFile(List<String> lines, String toFileName) {
```

Replace to the end of class